controllers/drcluster_mmode_test.go

```diff
@@ -144,6 +145,10 @@ var _ = Describe("DRClusterMModeTests", Ordered, func() {
 		k8sManager, err := ctrl.NewManager(cfg, options)
 		Expect(err).ToNot(HaveOccurred())
 
+		rateLimiter := workqueue.NewMaxOfRateLimiter(
+			workqueue.NewItemExponentialFailureRateLimiter(100*time.Millisecond, 100*time.Millisecond),
```

Comment:
Can we use smaller timeout? e.g. 10 ms?

Reply:
Do we really need 10ms, that is 10 reconciles (if each reconcile terminates within 10ms) and the 9 more in a second. IOW 19 reconciles a second (if there are errors or requeues).

For example till now VRG reconcile was 1s, with max at 1m, I am not sure we need to be more aggressive.

But I am not going to split hairs on this one, if you still feel it should be 10ms base, then I will just change it.

Reply:
Run output with 10ms base:
```
go test ./... -coverprofile cover.out
?   	github.com/ramendr/ramen	[no test files]
?   	github.com/ramendr/ramen/controllers/argocd	[no test files]
?   	github.com/ramendr/ramen/controllers/kubeobjects	[no test files]
?   	github.com/ramendr/ramen/controllers/kubeobjects/velero	[no test files]
ok  	github.com/ramendr/ramen/controllers	65.567s	coverage: 66.4% of statements
ok  	github.com/ramendr/ramen/controllers/cel	6.203s	coverage: [no statements]
ok  	github.com/ramendr/ramen/controllers/util	5.885s	coverage: 20.3% of statements
ok  	github.com/ramendr/ramen/controllers/volsync	19.441s	coverage: 57.6% of statements
```

Thought the second run failed with some VRG test case attempting to update a resource that is stale (`object has been modified`)

Reply:
No new errors in about ~30 reconciles...

Reply:
Changed to 10ms